iceoryx_hoofs/test/moduletests/test_design_newtype.cpp

```diff
@@ -76,48 +75,20 @@ struct Sut : public T
     Sut& operator=(Sut&&) noexcept = default;
 };
 
-CompileTest compileTest(R"(
-    #include \"iceoryx_hoofs/cxx/newtype.hpp\"
-
-    template <typename T>
-    struct Sut : public T
-    {
-        using T::T;
-        using T::operator=; // bring all operator= into scope
-    };
-
-    using namespace iox;
-    using namespace iox::cxx;
-)",
-                        {"iceoryx_hoofs/include"});
-
-
 TEST(NewType, ComparableDoesCompile)
 {
     ::testing::Test::RecordProperty("TEST_ID", "a2c2823b-3593-4d45-845d-fea249362f11");
-    using SutType = Sut<cxx::NewType<int, newtype::ConstructByValueCopy, newtype::Comparable>>;
+    using SutType = Sut<NewType<int, newtype::ConstructByValueCopy, newtype::Comparable>>;
     SutType a(123);
     SutType b(456);
     EXPECT_TRUE(a != b);
     EXPECT_FALSE(a == b);
 }
 
-#if !defined(_WIN32)
-TEST(NewType, NoComparableDoesNotCompile)
```

Comment:
This test doesn't compile but not for the right reason. It does not compile because the include path is not found, but the expected (only) reason should be the insufficient policy. This is already the case on master and I couldn't fix it. If you don't have an idea on how to fix this I would remove the test.

Reply:
see [#213](https://github.com/eclipse-iceoryx/iceoryx/issues/213)